service/routes.py

```diff
@@ -92,15 +96,41 @@ def get_recommendations(rec_id):
     app.logger.info("Request for recommendation with id: %s", rec_id)
     recommendation = Recommendation.find(rec_id)
     if not recommendation:
-        raise NotFound("Recommendation with id '{}' was not found.".format(rec_id))
+        raise NotFound(
+            "Recommendation with id '{}' was not found.".format(rec_id))
 
     app.logger.info("Returning recommendation: %s", recommendation.id)
     return make_response(jsonify(recommendation.serialize()), status.HTTP_200_OK)
 
+######################################################################
+# UPDATE AN EXISTING RECOMMENDATION
+######################################################################
+
+
+@app.route("/recommendations/<int:rec_id>", methods=["PUT"])
+def update_recommendations(rec_id):
```

Comment:
Can we put a list of all these URLs in the index?

In the json that is returned -> paths: [url_for(list...), url_for(update...)] etc.